tests/flow/test_load_csv.py

```
@@ -0,0 +1,225 @@
+def create_malformed_csv():
+    name   = MALFORMED_CSV
+    DATA   = MALFORMED_CSV_DATA
+    HEADER = MALFORMED_CSV_HEADER
+
+    create_csv_file(name, HEADER, DATA)
+
+    return name
+
+class testLoadCSV():
+    def __init__(self):
+        self.env, self.db = Env()
```

Ruff (0.8.2)

`Env` may be undefined, or defined from star imports

(F405)